records/config.go

```diff
@@ -177,14 +177,13 @@ func readConfig(file string) (*Config, error) {
 	c := NewConfig()
 
 	workingDir := "."
-	usr, err := user.Current()
-	if err != nil {
-		// this can happen (on Linux) if you're running mesos-dns as a non-root user.
-		logging.Error.Println("Failed to determine current user, translating ~/ to ./, error was", err)
-	} else {
-		workingDir = usr.HomeDir
+	for _, name := range []string{"HOME", "USERPROFILE"} { // *nix, windows
```

Comment:
http://stackoverflow.com/questions/606483/what-is-the-meaning-of-these-windows-enviroment-variables-homedrive-homepath